zebra-rpc/src/methods.rs

```diff
@@ -675,10 +675,12 @@ where
     }
 
     // TODO:
-    // - use a generic error constructor (#5548)
     // - use `height_from_signed_int()` to handle negative heights
     //   (this might be better in the state request, because it needs the state height)
     // - create a function that handles block hashes or heights, and use it in `z_get_treestate()`
+    // - Type second argument as an enum with variant for each verbosity level and a `Fields` variant
+    //   for specifying exactly which fields should be included in the return value. Consider replacing
+    //   `ReadRequest::BlockHeader` with a `PartialBlockData` request that accepts the fields?
```

Comment:
It would be nice to make this change in this PR to avoid always reading block headers while lightwalletd is syncing.